### PR TITLE
Rename :-webkit-autofill to :autofill

### PR DESCRIPTION
**Pull request by rachelandrew**
Working on https://github.com/mdn/content/issues/1725

Firefox has implemented `:autofill` adding the webkit prefixed version as an alias, as this is now part of a spec moving the property to the standard name with the webkit versions as prefixed versions.

### CODE REVIEW

**Review by jeanMS741 — APPROVED** [review]

**Review by jeanMS741 — APPROVED** [review]

File: css/selectors/autofill.json [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+              "version_added": "18",
+              "prefix": "-webkit-"
+            },
+            "edge": {
+              "version_added": "79",
+              "prefix": "-webkit-"
+            },
+            "firefox": {
+              "version_added": "86",
+              "notes": "Firefox has also implemented -webkit-autofill as an alias."
```
Comment (ddbeck) [inline css/selectors/autofill.json:23]: This note should be reported as two support statements in an array, one with a prefix and one without.

File: css/selectors/autofill.json [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+            },
+            "firefox_android": {
+              "version_added": "86",
+              "notes": "Firefox has also implemented -webkit-autofill as an alias."
```
Comment (ddbeck) [inline css/selectors/autofill.json:27]: As above, this note should be expressed as a separate support statement.

**Review by ddbeck — CHANGES_REQUESTED** [review]
One change needed to the Firefox data here. Thank you, @rachelandrew!

**Review by ddbeck — APPROVED** [review]
Thank you, @rachelandrew! 🎉

**Pull request merged**